src/chrome/create-chrome.ts

```diff
@@ -153,7 +155,7 @@ export const createChromeContext = ({
       }
       dispatch(toggleGlobalFilter(isHidden));
     },
-    isBeta,
+    isBeta: () => isPreview,
```

Comment:
Not sure if possible - but I wonder if we mark `isBeta` deprecated in the type and expose `isPreview` directly? I understand we need to keep this for some time though not to break consumers.

Reply:
Yep. We have to keep this, probably for ever same as we did with the chrome API on the window. We can update the TS in the types package once we rollout the change and have proper docs.